tools/logger/convert.c

```diff
@@ -68,7 +68,7 @@ static void log_err(FILE *out_fd, const char *fmt, ...)
 	if (buff) {
 		vsprintf(buff, fmt, args);
 		fprintf(stderr, "%s", buff);
-		if (out_fd) {
+		if (out_fd && out_fd != stderr && out_fd != stdout) {
```

Comment:
Could do with an inline comment here since we are testing three conditions.

Reply:
added